lib/inspec/schema.rb

```diff
@@ -147,6 +147,7 @@ class Schema
       'properties' => {
         'id' => { 'type' => 'string' },
         'profile_id' => { 'type' => %w{string null} },
+        'profile_sha256' => { 'type' => 'string' },
```

Comment:
I think `profile_sha256` communicates better and more clearly as to what the SHA is for.

When we then add things like: a id/'SHA' for the system, an id/sha for the 'run', etc. it will keep things more clear and seperated.